docs/yaml/functions/_build_target_base.yaml

```diff
@@ -328,3 +328,13 @@ kwargs:
 
       This allows renaming similar to the dependency renaming feature of cargo
       or `extern crate foo as bar` inside rust code.
+
+  rename:
+    type: list[str]
+    since: 1.6.99
```

Comment:
You should put `1.7` here. `.99` is the internal version number used for development. Same for all other occurences.

Reply:
Thanks; I've updated the two places in the code and the matching places in the docs. I used 1.7 in the docs although I'm seeing other entries adding the trailing .0. Is there any preference or convention I should respect here?

Reply:
Usually we use the full `X.Y.Z`. It's pretty rare now, but early on we would sometimes have things changed/added/removed in .Z releases.

Reply:
I'll add .0 then for historical interest